setup.cfg

```diff
@@ -4,6 +4,7 @@ universal = 1
 [flake8]
 max-line-length = 79
 exclude = */migrations/*, docs/conf.py
+ignore = E203, W503
```

Comment:
Why is E203 ignored?

Reply:
There are a lot of discussions about that in the web. Black decided to use a whitespace before and after a colon if it is used for slicing but flake8 complains about it even though it is in line with PEP8. [[Source]](https://github.com/PyCQA/pycodestyle/issues/373)

Reply:
I see, thank you!